index.html

```diff
@@ -76,6 +76,204 @@ <h4 class="title">I am a Software Developer</h4>
         </section>
         <!-- end of hero section -->
 
+        <!-- my recent works -->
+        <section class="recent-works-section">
+            <div class="heading-section">
+                <h3 class="recent-heading">My Recent Works</h3>
+                <div class="line"></div>
+            </div>
+            <div class="grid-container">
+                <div class="grid-items mp">
+                    <div class="grid-bg-img">
+                        
+                    </div>
+                    <div class="grid-content">
+                        <h3 class="grid-heading">Multi-Post Stories</h3>
+                        <p class="grid-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                        <ul class="tech-stack">
+                            <li class="tech-stack-li">CSS</li>
+                            <li class="tech-stack-li">HTML</li>
+                            <li class="tech-stack-li">Bootstrap</li>
+                            <li class="tech-stack-li">Ruby</li>
+                        </ul>
+                        <button class="see-projects">See Projects</button>
```

Comment:
- [OPTIONAL] Kindly fix the style of the languages list follow [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) .
Here you have a  screenshot of the list of how it is looking:
![image](https://user-images.githubusercontent.com/74506933/153720265-fc9edd88-c8a2-4abc-81e0-61c3d05847fa.png)

- [OPTIONAL] Kindly add hover of all cards buttons following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).